lib/file.js

```diff
@@ -150,8 +154,7 @@ async function downloadFileConcurrently(url, filepath, options) {
         downloadFiles: [
             {
                 fileUrl: url,
-                filePath: filepath,
-                fileSize: -1
+                filePath: filepath
```

Comment:
I was thinking of also adding the following in case we end up passing file size (`source.size` from nui) to this function since we do a head request in core

```suggestion
                filePath: filepath,
                fileSize: options && options.fileSize
```